src/mslice/cli/__init__.py

```diff
@@ -36,7 +36,7 @@ def pcolormesh(self, *args, **kwargs):
         else:
             return Axes.pcolormesh(self, *args, **kwargs)
 
-    def recoil(self, workspace, element=None, rmm=None):
+    def recoil(self, workspace, element=None, rmm=None, **kwargs):
```

Comment:
Can I ask why you decided to use `kwargs` here, and not just add an argument for color?

Reply:
Elsewhere in Mantid kwargs are used quite often to pass a large number of plotting options through the mantid interface and onto matplotlib. I decided that using kwargs here would keep consistency, and allows us to easily propagate more plotting arguments in the future (e.g. linewidth)